src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $-width);
+  gap: 48px;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media (max-width:1023px) {
+    grid-template-columns: repeat(3, $-width);
+  }
+
+  @media (max-width:767px) {
+    grid-template-columns: repeat(2, $-width);
+  }
+
+  @media (max-width:487px) {
+    grid-template-columns: $-width;
+  }
```

Comment:
```suggestion
.catalog {
  display: grid;
  justify-content: center;
  grid-template-columns: repeat(4, $-width);
  gap: 48px;
  
  
  @media (max-width:1023px) {
    grid-template-columns: repeat(3, $-width);
  }

  @media (max-width:767px) {
    grid-template-columns: repeat(2, $-width);
  }

  @media (max-width:487px) {
    grid-template-columns: $-width;
  }

  &__card {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 300px;
    border: 1px solid #ccc;
    border-radius: 5px;
  }
```

Reply:
Not critical, but better to keep media queries for particular block near to this block styles

Reply:
And as @mstrilec said, would be nice to create and use mixins for breakpoints